x-pack/plugin/esql/src/test/java/org/elasticsearch/xpack/esql/analysis/AnalyzerTests.java

```diff
@@ -1948,7 +1948,7 @@ public void testLookup() {
                  * Int is a reference here because we renamed it in project.
                  * If we hadn't it'd be a field and that'd be fine.
                  */
-                .item(containsString("int{r}"))
+                .item(containsString("int{f}"))
```

Comment:
This is subtle. Marking this for before we merge: if this change remains in place, I think we should update the comment above to make sense. Or we just disable the test for now but leave the initial expectations in place.